### PR TITLE
Show warning if browser does not support TextDecoder

### LINKED ISSUE #891: Error: Could not bootstrap application ReferenceError: 'TextDecoder' is not defined
When I open https://web.threema.ch in the Microsoft Edge Browser (Version 44.18362.1.0; Windows 10 Pro 1903 Build 18362.239) only the background image is visible and the dev tools console shows the following output:

```
CSP14309: Unbekannte Direktive block-all-mixed-content in Content-Security-Policy. Die Direktive wird ignoriert.
Compat: WebAssembly is supported
Could not bootstrap application ReferenceError: 'TextDecoder' is not defined
```

Can I provide any other information that would help you to fix this issue?

Thank you and best regards,
Christof Luick

### PR DESCRIPTION
**Pull request by dbrgn**
img

Will test this as soon as the 7 GiB Windows 10 Test-VM has finished downloading... :roll_eyes: 

Closes #891.

### CODE REVIEW

**Comment by ovalseven8** [comment]
Is localization enabled for this text?

**Comment by dbrgn** [comment]
> Is localization enabled for this text?

No, since the entire localization engine is not loaded at this point (it's part of the bundle, which cannot be loaded precisely because the browser is not compatible with certain features used inside the bundle).

File: index.html [hunk 1]
```diff
@@ -77,6 +77,16 @@ <h2>Error: Browser not supported</h2>
         </div>
     </div>
 
+    <div class="compat textdecoder" style="display: none;">
+        <img class="logo-compat" src="img/logo.svg?v=[[VERSION]]"/>
+        <div>
+            <h2>Error: Browser not supported</h2>
```
Comment (lgrahl) [inline index.html]: Super-nit: If I have a browser that supports neither, I'll get this heading twice now.

Comment (dbrgn) [reply index.html]: Yep, but I don't care all that much :wink:

**Comment by lgrahl** [comment]
@dbrgn: Status?

**Comment by dbrgn** [comment]
Good thing I tested it again. It didn't work :sweat_smile: 

Should be OK now, even with merged logo:

VirtualBox_IE11 - Win81_27_08_2019_11_37_45

**Comment by dbrgn** [comment]
And latest Edge (44):

VirtualBox_MSEdge - Win10_27_08_2019_12_08_59

**Comment by ovalseven8** [comment]
The coming Edge, which is based on Chromium, will then fully support Threema Web, right?

**Comment by dbrgn** [comment]
> The coming Edge, which is based on Chromium, will then fully support Threema Web, right?

Probably, yes.

**Comment by dbrgn** [comment]
I just tested it: Edge Preview is detected as Chrome 77 and works fine.

**Pull request merged**